test/test.js

```diff
@@ -4,44 +4,11 @@ var url = require('url');
 var expect = require('chai').expect;
 var objectAssign = require('object-assign');
 var format = require('string-format');
-var rimraf = require("rimraf");
-var fs = require('fs');
 var common = require('../lib/common')
 
-/*rimraf('sitespeed-result', function(err) {
-    console.log("delete legacy test data success")
-});
-*/
-
-var path1 = __dirname + '/../../../config/budget-testcases.json';
-var path2 = __dirname + '/../config/budget-testcases.json';
-var config = process.env.BUDGET_TESTCASES;
-if (!config) {
-    if (fs.existsSync(path1)) {
```

Comment:
Yeah, I understood your meaning about move bleumix related logic out of this package, here I just want let each caller load config file by themselves, so they can put all parameters in one config file(like username and password which is needed by cookie getting),  and caller still can add other parameter to this config object dynamically,  eg: add a request header after getting the cookie,  so it should be more flexible. Meanwhile, the whole process just need to load config file from one place, it makes logic of config file loading more simple as well.  what do you think?